src/main/webapp/js/userMapData.json

```diff
@@ -857,6 +904,7 @@
       "California State University Long Beach",
       "California State University Northridge",
       "California State University San Marcos",
+      "California State University, Stanislaus",
```

Comment:
Check if this is a real thing. If so, can remove the comma after `University`, to be consistent with the previous few lines.